multiplication_table.py

```diff
@@ -1,14 +1,9 @@
-from sys import argv # import argment variable 
+argv = input('Number of ROWS and COLUMBS: ')
 
-script, rows, columns = argv # define rows and columns for the table and assign them to the argument variable
+script, rows, columns = argv, argv, argv # define rows and columns for the table and assign them to the argument variable
 
 
 def table(rows, columns):
-	for i in range(1, int(rows) + 1 ): # it's safe to assume that the user would mean 12 rows when they provide 12 as an argument, b'coz 12 will produce 11 rows
-		print ("\t", i,)
-
-	print ("\n\n") # add 3 lines
-
 	for i in range(1, int(columns) + 1 ):
 		print (i),
```

Comment:
__print (i),__ --> __print(i, end="")__ # for Python 3

Reply:
Also put __from \_\_future\_\_ import print_function__ at the top of the file and use the line __print("\t",i*j, end="")__ near the bottom of the file.